src/app/app.routing.js

```diff
@@ -0,0 +1,23 @@
+import { RouterModule } from '@angular/router';
+
+export const routes = [
+    { path: '', redirectTo: 'dashboard', pathMatch: 'full', useAsDefault: true },
+    {
+        path: 'help',
+        loadChildren: () => require('es6-promise!./help/help.module')('HelpModule')
```

Comment:
it should load the module asynchronously, on demand. I saw 3 different approaches: system.import, angular 2 syntax and require with es6 promise loader. First and second didn`t work for me at all.